Automattic/wp-calypso
# Update/cancellation modal feature list

#### Proposed Changes
Add the Cancellation Feature List.
Return the list of features the user is going to loose cancelling their plan.

#### Feature flags:
URL query: `?flags=pre-cancellation-modal`
Shell command: `ENABLE_FEATURES=pre-cancellation-modal yarn start`

#### Design ( with features )
vTj2aknM6tRUQndWtOcURT-fi-291%3A4989

#### Notes:
- The subtitle and domain cancellation feature item are still placeholder 
- The refund placeholder at the bottom of dialog is addressed by the PR https://github.com/Automattic/wp-calypso/pull/69586

#### Testing Instructions
- Visit the Calypso purchases page:  http://calypso.localhost:3000/me/purchases/
- Test all plans and verify that the feature list matches the design. Note that in the comment https://github.com/Automattic/wp-calypso/pull/69276#issuecomment-1295566944 the number of elements has been restricted.

#### Screenshot:
<img width="692" alt="Screenshot 2022-11-02 at 06 28 59" src="https://user-images.githubusercontent.com/5706607/199405790-10f30ff4-88d9-4611-b328-ad945ef3da00.png">

#### Related to: [#1222](https://github.com/Automattic/martech/issues/1222)
